modules/issue_tracker/jsx/attachments/attachmentsList.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * React component used to display
+ * issue_tracker attachments list.
+ */
+import React, {Component, Fragment} from 'react';
+import PropTypes from 'prop-types';
+import Modal from 'jsx/Modal';
+
+class AttachmentsList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      attachments: this.props.attachments,
+      showModalAttachmentDelete: false,
+      deleteItem: {
+        ID: '',
+        file_name: '',
+        file_hash: '',
+      },
+    };
+    this.deleteAttachment = this.deleteAttachment.bind(this);
+    this.openModalAttachmentDelete = this.openModalAttachmentDelete.bind(this);
+    this.closeModalAttachmentDelete = this.closeModalAttachmentDelete.bind(this);
+    this.displayAttachmentOptions = this.displayAttachmentOptions.bind(this);
+  }
+
+  deleteAttachment() {
```

Comment:
It would be nice to add function comments to the JS files to prevent eslint warnings

Reply:
Thanks @johnsaigle, latest commit adds comments for that file.